README.md

```diff
@@ -43,15 +31,3 @@ You can use [EspExceptionDecoder](https://github.com/me-no-dev/EspExceptionDecod
 Before reporting an issue, make sure you've searched for similar one that was already created. Also make sure to go through all the issues labelled as [for reference](https://github.com/espressif/arduino-esp32/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3A%22for%20reference%22%20).
 
 Finally, if you are sure no one else had the issue, follow the [ISSUE_TEMPLATE](docs/ISSUE_TEMPLATE.md) while reporting any issue.
-
```

Comment:
what about adding Contributing section with link to Readthedocs project again to be sure that users are informed enough.